### PR TITLE
Configure pre-commit to check the code style

### LINKED ISSUE #9: Pre-commit to check the code style

### PR DESCRIPTION
**Pull request by workingloong**
Fix #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,31 +4,34 @@ EasyDL is a system to support elastic,
 fault-tolerance, automatic resource configuration and automating scaling
 for distributued deep learning jobs on the cloud.
```
Comment (nash635) [inline README.md:5]: distributued -> distributed

File: README.md [hunk 2]
```diff
@@ -4,31 +4,34 @@ EasyDL is a system to support elastic,
...
-DL training jobs on a Kubernetes (K8s) cluster. 
+DL training jobs on a Kubernetes (K8s) cluster.
 
 EasyDL consists three components:
+
 - ElasticTrainer: A framework to use EasyDL in training.
 - ElasticOperator: A k8s controller to manage training Pods.
-- Brain: An optimization service to generate resource
-configurations for training jobs.
+- Brain: An optimization service to generate resource.
```
Comment (nash635) [inline README.md:13]: resource -> resources

File: README.md [hunk 3]
```diff
@@ -4,31 +4,34 @@ EasyDL is a system to support elastic,
...
 ### Fault-tolerance
+
 EasyDL can recover failed parameter servers and workers and resume the training.
 Some failed nodes do not interrupt the training and hurts the convergence
 accuracy.
 
 ### Elaticity
+
 EasyDL can scale up/down the resources(CPU, memory and number) of workers
 and PS during training. Each node can have its own resource configuration
```
Comment (nash635) [inline README.md:34]: own -> remove

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+          (elasticdl/python/tests/data|easydl/go/vendor|easydl/go/manifests)
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.23.1
+    hooks:
+      - id: markdownlint
+        name: markdownlint
+        description: "Checks the style of Markdown files."
+        entry: markdownlint
+        language: node
+        exclude: (easydl/go/vendor|docs/ant_release)
```
Comment (nash635) [inline .pre-commit-config.yaml:78]: docs/ant_release 
这个开源版不应该有吧？

File: .isort.cfg [hunk 5]
```diff
@@ -0,0 +1,5 @@
+[settings]
+known_third_party = 
+multi_line_output=3
+line_length=79
```
Comment (nash635) [inline .isort.cfg:4]: line_length 改成120是不是好点

Comment (workingloong) [reply .isort.cfg:4]: The line_length follows [Google Python Style](https://google.github.io/styleguide/pyguide.html#3-python-style-rules ) where the maximum line length is 80 characters.

**Review by nash635 — APPROVED** [review]

**Pull request merged**